renovate-base.json5

```diff
@@ -3,6 +3,16 @@
   "extends": [
     "config:base"
   ],
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/bower_components/**",
+    "**/vendor/**",
+    "**/examples/**",
+    "**/__tests__/**",
+    "**/test/**",
+//    "**/tests/**", Disabling default ignored path
```

Comment:
Would it make sense to also disable the one above (`**/test/**`)?

Reply:
I am fine with it, actually very likely it is used somewhere